gateway/mw_graphql.go

```diff
@@ -17,17 +21,19 @@ func (m *GraphQLMiddleware) EnabledForSpec() bool {
 }
 
 func (m *GraphQLMiddleware) Init() {
-	logger := m.Logger()
-	logger.Info("I'm loaded")
-
-}
+	schema, err := gql.NewSchemaFromString(m.Spec.GraphQL.GraphQLAPI.Schema)
+	if err != nil {
+		log.Errorf("Error while creating schema from API definition: %v", err)
+	}
 
-func (m *GraphQLMiddleware) Destructor() {
+	m.Schema = schema
 }
 
 func (m *GraphQLMiddleware) ProcessRequest(w http.ResponseWriter, r *http.Request, _ interface{}) (error, int) {
-	logger := m.Logger()
-	logger.Info("I'm loaded")
+
+	if m.Schema == nil {
+		return errors.New("schema is not created"), http.StatusInternalServerError
```

Comment:
Will this error message be returned to the client? I would avoid disclosing this information to the API user.

Reply:
If there was a problem with schema, when user makes a request, it is returned to the user